### PR TITLE
Policy Compliance Checks: Fix typos and the panel title may be covered by dropdown arrow

### PR DESCRIPTION
**Pull request by eason9487**
### Changes proposed in this Pull Request:

This PR fixes three typos that were found when testing #1686.

### Detailed test instructions:

Check if the typos are replaced with the correct ones.

### CODE REVIEW

**Review by mikkamp — COMMENTED** [review]
Thanks for fixing those typos, just have one comment about the period which should be outside the brackets:
> email or phone number).

image
Just wondering about the text for the contact information. In my browser that renders under the dropdown arrow. Is there a way we can shorten that sentence and maybe move some of it's text into the dropdown text?

How about "Confirm your store's contact information is visible on your website"?

**Comment by eason9487** [comment]
@mikkamp, thanks for the review.

> Just wondering about the text for the contact information. In my browser that renders under the dropdown arrow. Is there a way we can shorten that sentence and maybe move some of it's text into the dropdown text?

Considering texts could be in other languages after applying the i18n function, the length of a text may still be too long to fit with the panel's available size, so I adjusted CSS to add the needed space for the dropdown arrow.
Kapture 2022-10-03 at 19 10 09

Could you help with a new round of code reviews?

File: js/src/setup-mc/setup-stepper/store-requirements/pre-launch-checklist/index.js [hunk 1]
```diff
@@ -180,7 +180,7 @@ const PreLaunchChecklist = ( props ) => {
 								) }
 							>
 								{ __(
-									'Allow your customers to contact you for product inquiries by including contact information on your website (i,e, contact us form, business profile link, social media, email or phone number.)',
+									'Allow your customers to contact you for product inquiries by including contact information on your website (i.e. contact us form, business profile link, social media, email or phone number.)',
```
Comment (mikkamp) [inline js/src/setup-mc/setup-stepper/store-requirements/pre-launch-checklist/index.js]:
```suggestion
									'Allow your customers to contact you for product inquiries by including contact information on your website (i.e. contact us form, business profile link, social media, email or phone number).',
```

**Review by mikkamp — APPROVED** [review]
Thanks for the quick change, that looks good to me the text wraps now to leave space for the arrow.
image

I'll go ahead and approve but I still left one suggestion about the period moving outside the brackets.

File: js/src/setup-mc/setup-stepper/store-requirements/pre-launch-checklist/index.js (hunk 1 above)
Comment (eason9487) [reply js/src/setup-mc/setup-stepper/store-requirements/pre-launch-checklist/index.js]: Ah, I missed this part in the previous review comment. Thanks!

**Pull request merged**